File: Topic Based Problems/01 Fundamentals/Beginner/18 Kilometers to Miles/Java/KilometerstoMiles.java
/**
 * @file KilometerstoMiles.java
 * @author SharifdotG
 * @brief Kilometers to Miles in Java
 * @version 1.0
 * @date 2023-09-24
 * 
 * Copyright (c) 2023
 */

import java.util.Scanner;

public class KilometerstoMiles {
    public static void main(String[] args) {
        double kilometers;
        
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the distance in Kilometers: ");
        kilometers = scanner.nextDouble();
        scanner.close();

        System.out.printf("Distance in Miles = %.2f%n", kilometers * 0.621371);
    }
}

File: Topic Based Problems/01 Fundamentals/01 Beginner/18 Kilometers to Miles/C/KilometerstoMiles.c
/**
 * @file KilometerstoMiles.c
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Kilometers to Miles in C
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <stdio.h>

int main() {
    double kilometers;

    printf("Enter the distance in Kilometers: ");
    scanf("%lf", &kilometers);

    printf("Distance in Miles = %.2lf\n", kilometers * 0.621371);

    return 0;
}
